index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Baloo+Da+2:wght@400;500;600;700;800&family=Josefin+Slab:ital,wght@0,300;0,400;0,500;1,300;1,400&family=Lato:ital,wght@0,300;1,300;1,400&family=Mulish:ital,wght@0,300;0,400;0,500;0,600;1,200;1,300;1,400;1,500&family=Poppins:ital,wght@0,200;0,300;1,100;1,200;1,300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Zero to Mastery</title>
+</head>
+<body>
+  <header>
+    <div class="container">
+			 <nav id="socila-contact" class="socila-contact">
+				<a href="#socila-contact">
+					<img class="facebook" src="imgs/facebook.svg" alt="facebook">
+				</a>
+				<a href="#socila-contact">
+					<img class="twitter" src="imgs/twitter.svg" alt="twitter">
+				</a>
+				<a href="#">English</a>
+				<a href="#">My Page</a>
+				<a href="#">Logout</a>
+			 </nav>
+			 <nav class="desktop-nav">
+				<div class="left-desk"> 
+				<a href="index.html">
+					 <img class="desktop-logo" src="imgs/logo.png" alt="">
```

Comment:
- [ ] [OPTIONAL] please place something appropriate in the `alt` attribute of  all `img` tags.